src/commands/buildCommands/trn.js

```diff
@@ -1,40 +1,60 @@
+const { promisify } = require('util');
 const child_process = require('child_process');
 const fs = require('fs');
 const path = require('path');
+const chalk = require('chalk');
 const mkdirp = require('mkdirp');
 
-const { paths, locales } = require('mwp-config');
+const { paths, locales, package: packageConfig } = require('mwp-config');
 const {
 	allLocalPoTrnsWithFallbacks$,
 	localTrns$,
 } = require('../txCommands/util');
 
 const MODULES_PATH = path.resolve(paths.repoRoot, 'src/trns/modules/');
 
+const writeFile = promisify(fs.writeFile);
```

Comment:
Doesn't writefile already have a callback setup? Why are we promisfying?

Reply:
Feels like an unneeded abstraction

Reply:
using a Promise-based interface allows us to use [`Promise.all` on line 40](https://github.com/meetup/mwp-cli/pull/49/files#diff-fcfd2051cfbbe9a94bcb1df44fd4a116R46) in order to orchestrate writing all the files we need concurrently and returning only when they are all written. Using the vanilla callback interface requires refactoring callers to use callbacks or manually construct a `new Promise`, which is more awkward than Promises start to finish.

Reply:
Gotcha!